ftw/sliderblock/browser/resources/styles.scss

```diff
@@ -1 +1,11 @@
 @include portal-type-font-awesome-icon(ftw-sliderblock-sliderblock, forward);
+
+.ftw-sliderblock-sliderblock .inneredit {
+  font-size: $font-size-large;
+  line-height: $font-size-large;
+  position: absolute;
+  right: 0;
+
```

Comment:
@jone Probably a white slightly opac background, with a dark icon would be the best. Now it's also hard to see on the image with house/tree